runtime/compiler/control/JITServerCompilationThread.cpp

```diff
@@ -148,13 +148,25 @@ outOfProcessCompilationEnd(
 
    JITServer::ServerMemoryState memoryState = computeServerMemoryState(compInfoPT->getCompilationInfo());
 
+   // Send methods requring resolved trampolines in this compilation to the client
+   std::vector<TR_OpaqueMethodBlock *> methodsRequiringTrampolines;
+   if (comp->getMethodsRequiringTrampolines().size() > 0)
+      {
+      methodsRequiringTrampolines.reserve(comp->getMethodsRequiringTrampolines().size());
+      for (auto it : comp->getMethodsRequiringTrampolines())
+         {
+         methodsRequiringTrampolines.push_back(it);
```

Comment:
No we don't, because we don't explicitly use an iterator here, but we use a range-based for loop, which gives us elements of the vector directly. 
The same method is used to iterate over `SerializedRuntimeAssumption` vector just a few lines above, so I used it here as well.

Reply:
Could you please double check? In the `SerializedRuntimeAssumption` code we `push_back(*it)`.

Reply:
We dereference `SerializedRuntimeAssumption` because `getSerializedRuntimeAssumptions` returns `TR::list<SerializedRuntimeAssumption *>`, but the vector we send to the client is `std::vector<SerializedRuntimeAssumption>`.

Reply:
ok